libs/shared/typescript/charts/.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": ["../../../../.eslintrc.json"],
+  "ignorePatterns": ["!**/*"],
+  "env": {
+    "browser": true
+  },
```

Comment:
Specify that the charts library is intended to run in browser environments, so browser-specific interfaces are defined, e.g. `HTMLDivElement`

Reply:
@hallieswan Do you know if the Apache ECharts library supports SSR? We could bump the priority of enabling SSR up before investing time using Apache ECharts.

Reply:
The [Apache ECharts documentation](https://apache.github.io/echarts-handbook/en/how-to/cross-platform/server/) indicates that the library supports SSR, but I'm not sure how much effort it would be to implement.

Reply:
We can test this support soon when we setup the project for SSR.